src/opentelemetry/ext/lightstep/__init__.py

```diff
@@ -10,7 +8,9 @@
 
 def _nsec_to_sec(nsec):
     """Convert nanoseconds to seconds float"""
-    return nsec / 1000000000
+    if nsec:
+        return nsec / 1000000000
+    return 0
```

Comment:
This can lead to some kind of type mismatch since this function may return an `int` or a `float`. Maybe it is better to do this check in the caller or to return a `float` always.